File: C#/1564 Vai Ter Copa.cs
using System;

class MainClass {
  public static void Main (string[] args) 
  {
    string s;
    while(!string.IsNullOrEmpty(s = Console.ReadLine()))
    {
      if (s == "0") Console.WriteLine("vai ter copa!");
      else Console.WriteLine("vai ter duas!");
    }
  }
}

File: C/1564 Vai Ter Copa.c
#include <stdio.h>

int main(void) 
{
  int a;
  while(scanf("%d" ,&a)!=EOF)
  {
    if(a == 0) printf("vai ter copa!\n");
    else printf("vai ter duas!\n");
  }
  return 0;
}
